### PR TITLE
Fuse dataset

### PR DESCRIPTION
**Pull request by Canwang-sjtu**

### CODE REVIEW

File: mmpose/datasets/pipelines/shared_transform.py [hunk 1]
```diff
@@ -43,6 +44,51 @@ def __call__(self, results):
...
+        self.keys = keys
+
+    def __call__(self, results):
+        num_output_channels = results['ann_info']['num_output_channels']
+        dataset_channel = results['ann_info']['dataset_channel']
+
+        target = results['target']
+        target_weight = results['target_weight']
+
+        # print(results['target'].shape, type(results['target']))
```
Comment (jin-s13) [inline mmpose/datasets/pipelines/shared_transform.py:68]: delete unnecessary comments

File: requirements/readthedocs.txt [hunk 2]
```diff
@@ -4,6 +4,6 @@ poseval@git+https://github.com/svenkreiss/poseval.git
 scipy
 smplx
 titlecase
-torch
-torchvision
+#torch
+#torchvision
```
Comment (innerlee) [inline requirements/readthedocs.txt:8]: unrelated change?
